Source/Tools/CommonTools/Linux/Utils/ELFContainer.cpp

```diff
@@ -69,7 +65,7 @@ ELFContainer::ELFType ELFContainer::GetELFType(int FD) {
   // We can't use dup since that makes the FD have the same underlying state backing both FDs.
 
   // We need to first determine the file size through fstat.
-  struct stat buf{};
+  struct stat buf {};
```

Comment:
It's adding a space here, but down below it uses `const char *RawString{};` (no space before `{}`).

Reply:
The difference here is that the `{}` after `const char *RawString` are parsed as a Cpp11BracedList and therefore affected by `SpaceBeforeCpp11BracedList`. Setting this option to true, adds a space after `RawString`.